QualityControl/public/pages/layoutListView/model/LayoutListModel.js

```diff
@@ -115,4 +115,9 @@ export default class LayoutListModel extends Observable {
       },
     };
   }
+
+  /**
+   * This function is required for all pages that have the filter element in their header
+   */
+  triggerFilter() {}
```

Comment:
Instead of adding empty methods, you could make use of the `extends` property of classes.
Something like
```js
export default class BaseViewModel extends Observable {
  /**
   * Default triggerFilter implementation to be overwritten by the subclasses
   */
  triggerFilter() {
  }
}
```

Use:
```js
export default class AboutViewModel extends BaseViewModel {
 [...] existing code without the empty triggerFilter
}
```